### PR TITLE
Eclipse's tinydtls added to CI test suite

### PR DESCRIPTION
**Pull request by pfg666**
Adding Eclipse's tinydtls was a bit tricky due to segmentation fault triggered by certain executions, which took a long time to register on a slow system. To address this, we compiled using address sanitization, which prevented these segmentation faults.

### CODE REVIEW

**Comment by pfg666** [comment]
Should be good to go. There is a concern about varying delay in detecting when the SUT terminates, which can lead to non-determinism.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -56,6 +56,60 @@ jobs:
...
+
+      - name: Install DTLS-Fuzzer
+        run: ./install.sh
+
+      - name: Setup SUT
+        run: ./setup_sut.sh etinydtls
+
+      - name: Test Eclipse TinyDTLS Server (psk)
+        run: |
+          java -jar target/dtls-fuzzer.jar args/etinydtls/learn_etinydtls_server_psk -roundLimit 5 -inputResponseTimeout CHANGE_CIPHER_SPEC:250
```
Comment (kostis) [inline .github/workflows/ci.yml:83]: Put the two `-roundLimit 5` _last_ on the two `java -jar ...` command`, for analogy with the other similar lines in this file and for maintaining the correspondence with the `5` in the line(s) below

File: experiments/results/servers/etinydtls_psk_rwalk/sul.config [hunk 2]
```diff
@@ -30,86 +36,53 @@
...
+            <group>SECP256R1</group>
+        </EcPrivateKey>
+        <signatureGroup>SECP256R1</signatureGroup>
+        <publicKeyGroup>SECP256R1</publicKeyGroup>
     </defaultExplicitCertificateKeyPair>
 </Config>
-148990152883696459058876040710756735692007315440292418837658355824640242244343067991959558873311625321072693673194440794503401720278880490501301726411910417932510993430937104762261186098410906851635860351964996874234449577481641189385117140763431243</modulus>
-            <privateExponent>54019007107306277718900683978022716693319969996336260
\ No newline at end of file
+    <EcPublicKey>
```
Comment (kostis) [inline experiments/results/servers/etinydtls_psk_rwalk/sul.config:74]: Something seems messed up below this line (and definitely in the last ones...)

File: setup_sut.sh [hunk 3]
```diff
@@ -482,7 +482,9 @@ function make_sut() {
             PKG_CONFIG_PATH="$MODULES_DIR/$nettle:$PKG_CONFIG_PATH" ./configure --enable-static --with-guile-site-dir=no --with-included-libtasn1 --with-included-unistring --without-p11-kit --disable-guile --disable-doc
         )
     elif [[ $sut == etinydtls* ]]; then
-        ( cd $sut_dir ; autoconf ; autoheader ; ./configure )
+        # for ETinyDTLS, we need the CC and CPP to be set to absolute paths (to avert Permission Denied when running make)
+        # we also enable address sanitization to prevent segmentation faults, which can take a long time to register and can be triggered in the current implementation
```
Comment (kostis) [inline setup_sut.sh:486]: This comment (about enabling address sanitization) is very cryptic and needs to be improved (or revised).
But we can discuss this issue in person.

File: experiments/results/servers/etinydtls_psk_rwalk/sul.config (hunk 2 above)
Comment (pfg666) [reply experiments/results/servers/etinydtls_psk_rwalk/sul.config:74]: There is a (more general) bug when exporting the TLS-Attacker configuration. This affects all experiments in the results folder. I will change the sul.config, but the bug itself will need to be addressed as a separate issue.

**Review by kostis — APPROVED** [review]

**Pull request merged**